src/components/icon/library.system.ts

```diff
@@ -86,11 +86,10 @@ const icons = {
 const systemLibrary: IconLibrary = {
   name: 'system',
   resolver: (name: keyof typeof icons) => {
-    if (icons[name]) {
+    if (name in icons) {
```

Comment:
This is the same reason as the other explicit boolean checks. Basically anywhere where a condition expects a true/false, it must be `true` or `false`, not something that is truthy. And the `icons[name]` check is a truthy cooercion. If the value was an empty string, it'd be false for example. What I believe the intent was is to check if the key exists in the object so that was the reason for the change.